src/giskardpy/god_map.py

```diff
@@ -63,11 +63,14 @@ class GodMap:
     eq_constraints: Dict[str, EqualityConstraint]
     neq_constraints: Dict[str, InequalityConstraint]
     derivative_constraints: Dict[str, DerivativeInequalityConstraint]
+    manip_constraints: Dict[str, ManipulabilityConstraint]
     hack: float
     fill_trajectory_velocity_values: bool
     ros_visualizer: ROSMsgVisualization
     free_variables: List[FreeVariable]
     controlled_joints: List[Joint]
+    m_index: List[float]
+    manip_gain: float
```

Comment:
not sure they should be direct god_map members. Wouldn't they make more sense as qp controller attributes?